src/test/suite/IntegrationUtils.test.ts

```diff
@@ -55,12 +58,24 @@ suite("IntegrationUtil tests", function() {
         showQuickPickStub.onFirstCall().returns(IntegrationUtils.vscodeTasksIntegration);
         showQuickPickStub.onSecondCall().returns(undefined);
         
-        await IntegrationUtils.startIntegration(getDocUri('ADifferentRouteBuilder.java'));
+        await IntegrationUtils.startIntegration(getDocUri('ADifferentRouteBuilder.java')).catch(() => {});
```

Comment:
Better use 
`
try {
    await IntegrationUtils.startIntegration(getDocUri('ADifferentRouteBuilder.java'));
} catch( (err) => {
    // and do something with the catched error.
});